### DISCUSSION TITLE
请问通过stream load导入数据,字段含有中文报错too many filtered rows怎么处理

### DISCUSSION THREAD

**Discussion by bienao666** [apache/doris#7416] [A - General / Q&A]
Reason: column(InsideId) value is null while columns is not nullable. src line:

**Comment by aihai** [comment]
You can refer to `max_filter_ratio` http://doris.incubator.apache.org/master/zh-CN/administrator-guide/load-data/stream-load-manual.html#%E5%AF%BC%E5%85%A5%E4%BB%BB%E5%8A%A1%E5%8F%82%E6%95%B0

The table schema should be compatible with the data.

**Comment by hf200012** [comment]
There is a problem with your data, you can check it through the errorurl in the return message
